docs/v0.5/en/user-guide/cluster.md

```diff
@@ -53,7 +53,7 @@ You can follow the steps to use SQL to play with distributed insertions and quer
    Query OK, 3 rows affected (0.09 sec)
    ```
 
-   The `dist_table` is distributed among the `Datanode`s. You can refer to ["Table Sharding"](https://docs.greptime.com/contributor-guide/frontend/table-sharding) for more details.
```

Comment:
needs to be updated, thanks

Reply:
What does it mean? I checked this link locally, and it works well. To what content you'd like to update?

Reply:
sorry, I mean the change is great 👍